components/landing/SearchLanding.vue

```diff
@@ -22,8 +22,8 @@
           v-for="chain in chainList"
           :key="chain.value"
           :class="[
-            'mr-2 mb-2',
-            'chain-option',
+            'mr-4 mb-2',
+            'chain-option active',
```

Comment:
everything will be `active` and `active active` if selected?
plus I find it more practical to change `mr-4 mb-2` with `m-2`